server/bin/gold/test-18.txt

```diff
@@ -6,25 +6,26 @@ md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001
 
 
 cp -a /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz.md5 /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/
 md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz.md5
 ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/BACKED-UP/tarball_config-02_1970.01.01T00.00.41.tar.xz
 ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/TO-RE-UNPACK/tarball_config-02_1970.01.01T00.00.41.tar.xz
 
-Restored 2 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz
+Restored 2 (0 existing) of (4): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz
 
 
 cp -a /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5 /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/
 md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5
 ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/BACKED-UP/tarball_config-00_1970.01.01T00.00.40.tar.xz
 ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/TO-RE-UNPACK/tarball_config-00_1970.01.01T00.00.40.tar.xz
 
-Restored 3 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz
+Restored 3 (0 existing) of (4): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz
 
-Restored 3 tar balls from backup volume, /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup (ended at: 42.0, elapsed (0.00)
+Exists 1 (3 restored) of (4): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-exists_1970.01.01T00.00.39.tar.xz
+Restored 3 tar balls from backup volume, /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup (ended at: 42.0, elapsed: 0.00)
```

Comment:
This message suggests that it restored __three__ tarballs, but the directory listing below shows only *one* addition...is that correct?

Reply:
Thoughts, @portante?

Reply:
That is because `test-18` is the dry-run, and the counts are what _would_ have happened, where as the directory listing below show what did happen.